scripts/oss-fuzz.sh

```diff
@@ -57,3 +59,19 @@ $CC $CFLAGS -I"$DESTDIR/usr/include" -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -c -o
 $CXX $CXXFLAGS $LIB_FUZZING_ENGINE secilc-fuzzer.o "$DESTDIR/usr/lib/libsepol.a" -o "$OUT/secilc-fuzzer"
 
 zip -r "$OUT/secilc-fuzzer_seed_corpus.zip" secilc/test
+
+## checkpolicy fuzzer ##
+
+make -C checkpolicy clean
+make -C checkpolicy V=1 CFLAGS="$CFLAGS -I$DESTDIR/usr/include" -j"$(nproc)" checkobjects
```

Comment:
Looking at all those "-I$DESTDIR/usr/include" flags sprinkled all over the script (mostly by me (sorry about that)), I think by analogy with the other build scripts like `.github/workflows/run_tests.yml` it would probably make sense to just append it to CFLAGS/CXXFLAGS somewhere at the beginning of the script and be done with it.

Reply:
Modified `$CFLAGS` at the top of the script.